smart_open/s3.py

```diff
@@ -186,8 +187,7 @@ def seek(self, position):
             #
             self._body = io.BytesIO()
         else:
-            getkwargs = dict(Range=range_string)
-            self._body = _get(self._object, self._version_id, **getkwargs)['Body']
+            self._body = _get(self._object, self._version_id, Range=range_string)['Body']
```

Comment:
The _get function can raise a ClientError. We should handle it the same way everywhere (wrap in IOError).